src/graph/GroupByExecutor.cpp

```diff
@@ -403,7 +403,7 @@ StatusOr<std::unique_ptr<InterimResult>> GroupByExecutor::setupInterimResult() {
     if (rsWriter != nullptr) {
         result->setInterim(std::move(rsWriter));
     }
-    return result;
```

Comment:
Interesting.

Reply:
This is due to some subtle problem of GCC 7 or earlier.